LSP.sublime-settings

```diff
@@ -7,6 +7,9 @@
 
   // --- Document Formatting ------------------------------------------------------------
 
+  // Run the server's documentRangeFormattingProvider (if supported) on the pasted text.
```

Comment:
I see that this comment was formulated similar to the next comment. But I think it would me more user friendly to reword it without using the capability names from the specs (documentRangeFormattingProvider). "If supported, format the range of the pasted text"

And for the next setting: "Format a file before saving"